src/main/webapp/app/instructor-dashboard/instructor-dashboard.html

```diff
@@ -1,9 +1,12 @@
 
 <h2>{{$ctrl.exercise.title}} <small>{{$ctrl.results.length}} results</small></h2>
-
-
+<jhi-alert></jhi-alert>
 <div>
     <div class="button-toolbar pull-right">
+        <!--ng-click="$ctrl.buildPlanCleanup()"-->
+        <button class="btn btn-primary btn-sm" ui-sref="instructor-dashboard.buildplans-delete">
+            <i class="fa fa-eraser" aria-hidden="true"></i>&nbsp;Build plans cleanup
```

Comment:
Please call it "Cleanup build plans" and use the translation feature

Reply:
Done.